x-pack/metricbeat/include/list.go

```diff
@@ -15,6 +15,7 @@ import (
 	_ "github.com/elastic/beats/x-pack/metricbeat/module/aws/s3_daily_storage"
 	_ "github.com/elastic/beats/x-pack/metricbeat/module/aws/s3_request"
 	_ "github.com/elastic/beats/x-pack/metricbeat/module/aws/sqs"
+	_ "github.com/elastic/beats/x-pack/metricbeat/module/aws_light"
```

Comment:
@jsoriano question about the list.go, should `_ "github.com/elastic/beats/x-pack/metricbeat/module/cockroachdb/status"` be included in this list.go as well?